python_modules/dagster/dagster/_utils/alert.py

```diff
@@ -49,6 +50,7 @@ def _default_failure_email_subject(context) -> str:
 
 def send_email_via_ssl(
     email_from: str,
+    email_user: str,
```

Comment:
apologies for the delayed response. this may break existing users as it's a required field. any chance to make it optional?

Reply:
Yes, just pushed an update to handle that.  Thanks.